### PR TITLE
CRPL-2876: Adds AT query executor & query builder nodes

### PR DESCRIPTION
**Pull request by amatiszcr**

### CODE REVIEW

File: apis/airtable/query builder/query record count setter/1b357c49-e7a8-4610-bfd4-7260c481d9d4.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      },
+      "50f5307f-27a3-4be9-87db-39041e183708": {
+        "type": "number",
+        "name": "record count",
+        "description": "Receives the number of records to return."
+      }
+    },
+    "outputs": {
+      "e055c747-523c-495e-b6cb-c4fddc083f42": {
+        "type": "{\n  optional \"fields\": string[],\n   optional \"filterByFormula\": string,\n   optional \"sort\": \n    {\n      \"field\": string,\n      \"direction\": (\"desc\" or \"asc\")\n    }[],\n    optional \"maxRecords\": number,\n    optional \"pageSize\": number,\n    optional \"view\": string,\n    optional \"returnFieldsByFieldId\": boolean\n}",
```
Comment (lvass74) [inline apis/airtable/query builder/query record count setter/1b357c49-e7a8-4610-bfd4-7260c481d9d4.json:56]: Wouldn't be simpler to refer to the `query` input port's type here?  Like typeof `query`?
Generally for all builder prototypes...

File: apis/airtable/query executor/url builder/d1b92b70-8246-4872-a133-15a060908e3a.json [hunk 2]
```diff
@@ -0,0 +1,84 @@
+{
+  "name": "apis/airtable/Query executor/URL builder",
+  "interface": {
+    "inputs": {
+      "3399362b-b1ac-486a-9c32-b3de94b49d56": {
+        "type": "typeof `base ID` of `build table url`",
+        "name": "base ID",
+        "description": "[Inherited from port `base ID` of `build table url`] \nReceives the ID of an AirTable base.\n\nExample:\n```\n\"app...\",\n```"
```
Comment (lvass74) [inline apis/airtable/query executor/url builder/d1b92b70-8246-4872-a133-15a060908e3a.json:8]: Review or clear

File: apis/airtable/query executor/url builder/d1b92b70-8246-4872-a133-15a060908e3a.json [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+      },
+      "8465721c-6784-4b2c-af80-b4578a6e9b6a": {
+        "type": "typeof `table name` of `build table url`",
+        "name": "table name",
+        "description": "[Inherited from port `table name` of `build table url`] \nReceives a table's identifier relative to an AirTable base.\n\nExample:\n```\n\"...\",\n```"
```
Comment (lvass74) [inline apis/airtable/query executor/url builder/d1b92b70-8246-4872-a133-15a060908e3a.json:13]: Review or clear

File: apis/airtable/query executor/url builder/d1b92b70-8246-4872-a133-15a060908e3a.json [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+      }
+    },
+    "outputs": {
+      "6182f006-1f08-48aa-8654-ae16b4f1c190": {
+        "type": "typeof `concatenated` of `concat /listRecords`",
+        "name": "url",
+        "description": "[Inherited from port `concatenated` of `concat /listRecords`] \nSends the concatenation of the two inputs."
```
Comment (lvass74) [inline apis/airtable/query executor/url builder/d1b92b70-8246-4872-a133-15a060908e3a.json:20]: Review or clear

File: apis/airtable/query sort adder/group/46413ac8-08f7-4112-a959-4c1959971e68.json [hunk 5]
```diff
@@ -0,0 +1,118 @@
+{
+  "name": "apis/airtable/Query sort adder/Group",
```
Comment (lvass74) [inline apis/airtable/query sort adder/group/46413ac8-08f7-4112-a959-4c1959971e68.json:2]: Any better name maybe?

File: apis/airtable/query sort adder/group/46413ac8-08f7-4112-a959-4c1959971e68.json [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+  "interface": {
+    "inputs": {
+      "eb8ac420-dddf-4d51-ad04-10597f249f46": {
+        "type": "typeof `dict` of `has key tester`",
+        "name": "query",
+        "description": "[Inherited from port `dict` of `has key tester`] \nReceives the dictionary to test.\n\nExample:\n{ \"first\": 1, \"third\": 3, \"fifth\": 5 }"
```
Comment (lvass74) [inline apis/airtable/query sort adder/group/46413ac8-08f7-4112-a959-4c1959971e68.json:8]: Review or clear

File: apis/airtable/query sort adder/group/46413ac8-08f7-4112-a959-4c1959971e68.json [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+      }
+    },
+    "outputs": {
+      "9d12e972-712d-4cdc-9fb0-6b8881219fdf": {
+        "type": "typeof `data` of `default`",
+        "name": "sorts",
+        "description": "[Inherited from port `data` of `default`] \nSends store contents."
```
Comment (lvass74) [inline apis/airtable/query sort adder/group/46413ac8-08f7-4112-a959-4c1959971e68.json:15]: review or clear

File: data/array/type tester fork/19aa85fb-32fd-4877-a6c8-96aa1865e89e.json [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+          ],
+          "name": "type tester",
+          "prototype": "35c45234-1f7a-41c4-a876-48a6395770b8"
+        },
+        "97596dba-0362-4e27-ac60-a2091fbf35dd": {
+          "coordinates": [
+            44,
+            46.5
+          ],
+          "name": "fork B",
```
Comment (lvass74) [inline data/array/type tester fork/19aa85fb-32fd-4877-a6c8-96aa1865e89e.json:40]: better node name?

File: data/array/type tester fork/19aa85fb-32fd-4877-a6c8-96aa1865e89e.json [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+            "97596dba-0362-4e27-ac60-a2091fbf35dd": {
+              "26b011f0-6b5a-45a3-a4b6-a90d93ce7046": {}
+            }
+          }
+        }
+      }
+    }
+  },
+  "iconId": "802b6352-2279-4681-84b8-6558e185fd36",
+  "description": "Forwards input data to one of two outputs, depending on whether the input data is a an array.",
```
Comment (lvass74) [inline data/array/type tester fork/19aa85fb-32fd-4877-a6c8-96aa1865e89e.json:80]: typo "data is a an array"

**Review by lvass74 — CHANGES_REQUESTED** [review]

File: apis/airtable/query builder/query record count setter/1b357c49-e7a8-4610-bfd4-7260c481d9d4.json (hunk 1 above)
Comment (amatiszcr) [reply apis/airtable/query builder/query record count setter/1b357c49-e7a8-4610-bfd4-7260c481d9d4.json]: Changed to more specifics instead

File: apis/airtable/paging query executor/aggregator/58669382-e18c-45a8-874a-4481459cb097.json [hunk 10]
```diff
@@ -0,0 +1,146 @@
...
+      "dffd3be3-5fd3-4ebd-889d-0d57667c18db": {
+        "type": "typeof `data` of `aggregate data`",
+        "name": "data",
+        "description": "[Inherited from port `data` of `aggregate data`] \nReceives data items to be aggregated."
+      }
+    },
+    "outputs": {
+      "f8253e9a-d661-4751-8572-2c0f1fd5dd36": {
+        "type": "any",
+        "name": "aggr data"
```
[the remark this answers is not in the document]
Comment (amatiszcr) [reply apis/airtable/paging query executor/aggregator/58669382-e18c-45a8-874a-4481459cb097.json:23]: Prefer to leave it as it's unreadable. Can be changed later if we have that feature.
Abbreviations are widely used elsewhere, eg. 'dict'

**Review by lvass74 — APPROVED** [review]

**Pull request merged**